proxy/proxy/salt-broker/broker

```diff
@@ -33,3 +33,22 @@
 # default: 1 to send logs to file
 # 0 to send logs to standard output/error
 log_to_file: 1
+
+######   ZeroMQ connection options    ######
+############################################
+# Documentation notes: TBD
```

Comment:
I guess we need to add some text here. I'll continue with the review of the PR and will add the notes afterward.

Reply:
sure, makes sense :+1: